docs/install-and-setup.md

```diff
@@ -210,13 +210,17 @@ To set up command-line completion, source the output of
 `jj util completion bash/zsh/fish`. Exactly how to source it
 depends on your shell.
 
-Improved completions are currently in the works, these will complete things like
-bookmark names as well. You can activate them with the alternative "dynamic"
+Improved completions are also available, they will complete things like
+bookmarks, aliases, revisions, operations and files as well. They can be
```

Comment:
It took me a few seconds to figure out what "as well" refers to. I assume it's "as well as what the static completions do". I'm not sure we need to say that. What do you think about just deleting "as well"? Oh, we already say further down that they also support the static completions.